### ISSUE TITLE
Is PassFault compatible with java 1.6

### ISSUE THREAD

**Issue by PriyankaNaikwade** [OWASP/passfault#74]
Hi Team,

we want to use this library in our application so could dou please let us know Is is it compatible with java 1.6.

**Comment by PriyankaNaikwade** [comment]
HI Team,

Could you please update on this issue?

**Comment by c-a-m** [comment]
No.  About two years back I refactored to use Java 8 language features, so down compiling won't be possible. And we've added enough useful features since then that I wouldn't recommend going back to that commit and building from there.
